ingest/ledgerbackend/captive_core_backend_test.go

```diff
@@ -131,6 +133,9 @@ type testLedgerHeader struct {
 }
 
 func TestCaptiveNew(t *testing.T) {
+	storagePath, err := os.MkdirTemp(os.TempDir(), "captive-core-*")
```

Comment:
Fyi if you use the empty steering it will use the temp for by default. The first param is optional.
```suggestion
	storagePath, err := os.MkdirTemp("", "captive-core-*")
```